auction-server/Cargo.toml

```diff
@@ -49,6 +49,7 @@ bincode = "1.3.3"
 serde_with = { version = "3.9.0", features = ["base64"] }
 anchor-lang = "0.30.1"
 express-relay = { path = "../contracts/svm/programs/express_relay" }
+solana-rpc-client = "2.0.13"
```

Comment:
I don't think we need this explicitly since `solana-client` already exposes what we want:
`solana_client::rpc_sender::RpcSender`